source/docs/options/ja.markdown

```diff
@@ -0,0 +1,358 @@
+Brakemanのオプションドキュメント（[OPTIONS.md](https://github.com/presidentbeef/brakeman/blob/master/OPTIONS.md)）を翻訳しました。
+
+共同翻訳者は@jnchitoです。
+
+<hr>
+
+このファイルの情報は最新ではないかもしれません。最新の情報を知りたい場合は`brakeman --help`を実行してください（ただし、そこまで詳しくないかもしれません）。
```

Comment:
@gazayas Sorry for late feedback, but I think the lines above this line are not necessary, so could you remove them? I feel strange when I open this page: https://brakemanscanner.org/docs/options/ja

Reply:
Sure! I'll open a new pull request